modules/sharedaddy/sharing-service.php

```diff
@@ -749,7 +749,7 @@ function sharing_display( $text = '', $echo = false ) {
 	}
 
 	// Allow to be used on P2 ajax requests for latest posts.
-	if ( defined( 'DOING_AJAX' ) && DOING_AJAX && isset( $_REQUEST['action'] ) && 'get_latest_posts' == $_REQUEST['action'] ) {
+	if ( defined( 'DOING_AJAX' ) && DOING_AJAX && isset( $_REQUEST['action'] ) && apply_filters('sharing_ajax_action', 'get_latest_posts') == $_REQUEST['action'] ) {
```

Comment:
Could you split this into separate lines, and add a docblock as it's done here?
https://github.com/Automattic/jetpack/blob/216c8378ef24d6a1569232c2498767970a942872/modules/sharedaddy/sharing-service.php#L90

We use the docblock to automatically generate pages in the Jetpack codex:
https://developer.jetpack.com/